test/TesterInternal/ClientConfig_AzureStreamProviders.xml

```diff
@@ -16,4 +16,10 @@
     <Provider Type="Orleans.Providers.Streams.SimpleMessageStream.SimpleMessageStreamProvider" Name="SMSProvider" FireAndForgetDelivery="false"/>
     <Provider Type="Orleans.Providers.Streams.AzureQueue.AzureQueueStreamProvider" Name="AzureQueueProvider" DataConnectionString="UseDevelopmentStorage=true"/>
   </StreamProviders>
+  <Assemblies>
+    <Assembly Path=".\TestGrainInterfaces.dll"/>
+    <Assembly Path=".\TestFSharpInterfaces.dll"/>
+    <Assembly Path=".\Orleans.dll"/>
+    <Assembly Path=".\TestInternalGrainInterfaces.dll"/>
+  </Assemblies>
```

Comment:
For both cases it is just old code. You can use the assembly name and add only the root after our last iteration. I'll change it on those files.